cni-plugin/main.go

```diff
@@ -98,6 +99,21 @@ func configureLoggingLevel(logLevel string) {
 	}
 }
 
+// Use a MultiWriter to write both to stderr and to a logfile if
+// the environment variable LINKERD_CNI_DEBUG_LOGFILE is passed in.
+func setupDebugLog() {
+	if filename, ok := os.LookupEnv("LINKERD_CNI_DEBUG_LOGFILE"); ok {
```

Comment:
I suspect this should be more aggressively named to avoid people using it in production without log rotation.